tox.ini

```diff
@@ -104,3 +104,11 @@ commands =
     coverage html -d {toxworkdir}/htmlcov -i
     python -m diff_cover.diff_cover_tool --compare-branch {env:DIFF_AGAINST:origin/main} {toxworkdir}/coverage.xml
 depends = {py312, py311, py310, py39, py38, py37, pypy37, pypy38, pypy39}{,-min}, path
+
+[testenv:bump]
+description = bump versions, pass major/minor/patch
+skip_install = true
+setenv =
+deps = bump-my-version
```

Comment:
Add lower bound version spec

Reply:
Okay with something recent? Given it's internal, I'd assume yes. (0.10 came out yesterday I think)